--- a/SuiteSparse_config/SuiteSparse_config.mk	2018-12-23 03:15:58.000000000 +0000
+++ b/SuiteSparse_config/SuiteSparse_config.mk	2019-02-28 05:53:50.775276000 +0000
@@ -447,11 +447,14 @@
 SO_OPTS = $(LDFLAGS)
 
 ifeq ($(UNAME),Windows)
-    # Cygwin Make on Windows (untested)
+    # Cygwin Make on Windows
     AR_TARGET = $(LIBRARY).lib
-    SO_PLAIN  = $(LIBRARY).dll
+    SO_TARGET  = $(LIBRARY).dll
+    # The following two links are just garbage copies of the real target
+    # they aren't actually supported by this OS
     SO_MAIN   = $(LIBRARY).$(SO_VERSION).dll
-    SO_TARGET = $(LIBRARY).$(VERSION).dll
+    SO_PLAIN = $(LIBRARY).$(VERSION).dll
+    SO_OPTS  += -shared
     SO_INSTALL_NAME = echo
 else
     # Mac or Linux/Unix
